www/js/tablet.js

```diff
@@ -455,18 +423,23 @@ function tabletShowMessage(msg, collecting) {
     return;
   }
   if (msg.startsWith('error:')) {
-    const msgExtra = {
-      "8": " - Command requires idle state. Unlock machine?",
-      "152": " - Configuration is invalid. Maslow.yaml file may be corrupt. Try restarting",
-      "153": " - Configuration is invalid. ESP32 probably did a panic reset. Config changes cannot be saved. Try restarting",
-    };
-
-    msg += msgExtra[msg.split(":")[1]] || "";
+    let parts = msg.split(":");
```

Comment:
It looks like we might have the issue where some of these changes are undoing changes on the main branch. @md8n cleaned up some of this code here

Reply:
I'll have another look at my diffs and resolve this. I just noticed the merge I did had issues when I ran it so likely I just resolved things wrong.

Reply:
That always gets tricky